packages/react-composites/src/composites/common/Drawer/MoreDrawer.tsx

```diff
@@ -319,26 +337,30 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
         itemKey: 'dynamicSelectionKey',
         text: localeStrings.strings.call.moreButtonGalleryFloatingLocalLayoutLabel,
         onItemClick: () => {
+          /* @conditional-compile-remove(gallery-layout-composite) */
```

Comment:
this looks unrelated to reactions - was this needed for a build fix?

Reply:
Yes. Otherwise it wasn't passing the CI job checks.

Reply:
This is a GA'd feature part of gallery layouts

Reply:
But there are other parts where we have cc ... those were not removed. so it ends up bubbling up errors. For the sake of successful I am adding this. Removing them should be a different KTLO item.

Reply:
Thanks @dmceachernmsft , looks like removing line 21 and others solved the issue. Removing them from all occurances.